### PR TITLE
fix(dashboard): better external event UX

### PR DESCRIPTION
**Pull request by HazimAr**
- Timeouts are now shown
- Timeouts are formatted 165 seconds -> "2m 45s"
- `Inspect TaskRun` -> `Inspect ExternalEvent`
- `Inspect ExternalEvent` only shown once `ExternalEvent` has been called

# Timeouts:
image
image

### CODE REVIEW

File: dashboard/src/app/(authenticated)/externalEventDef/[name]/components/ExternalEventDef.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ export const ExternalEventDef: FC<Props> = ({ spec }) => {
   const { tenantId } = useWhoAmI()
 
   const { isPending, data, hasNextPage, fetchNextPage } = useInfiniteQuery({
-    queryKey: ['taskRun', tenantId, 10, createdAfter, createdBefore, isClaimed],
+    queryKey: ['externalEvent', tenantId, 10, createdAfter, createdBefore, isClaimed],
```
Comment (mijailr) [inline dashboard/src/app/(authenticated)/externalEventDef/[name]/components/ExternalEventDef.tsx:31]: What that 10 means?

**Review by mijailr — CHANGES_REQUESTED** [review]
Why is it renaming userTask?

File: dashboard/src/app/(authenticated)/externalEventDef/[name]/components/ExternalEventDef.tsx (hunk 1 above)
Comment (HazimAr) [reply dashboard/src/app/(authenticated)/externalEventDef/[name]/components/ExternalEventDef.tsx:31]: Not sure, ill remove it, it was copied from somewhere else. Maybe its needed in that other example so I'm not going to remove it in the other instance.

**Comment by HazimAr** [comment]
> Why is it renaming userTask?

Good catch, looks like I accidentally moved the folder.

File: dashboard/src/app/utils/date.ts [hunk 2]
```diff
@@ -14,3 +14,28 @@ export const utcToLocalDateTime = (utcISODateTime: string): string =>
   new Date(Date.parse(utcISODateTime)).toLocaleString(undefined, { hour12: false, timeZoneName: 'short' })
 
 export const localDateTimeToUTCIsoString = (localDateTime: string): string => new Date(localDateTime).toISOString()
+
+export const formatTime = (seconds: number): string => {
```
Comment (mijailr) [inline dashboard/src/app/utils/date.ts:18]: Can you add unit tests for this?

**Review by mijailr — CHANGES_REQUESTED** [review]

**Review by mijailr — APPROVED** [review]

**Pull request merged**